receiver/opencensusreceiver/opencensus_test.go

```diff
@@ -153,7 +153,7 @@ func TestGrpcGateway_endToEnd(t *testing.T) {
 	assert.True(t, proto.Equal(want.Resource, gotOc[0].Resource))
 	require.Len(t, want.Spans, 1)
 	require.Len(t, gotOc[0].Spans, 1)
-	assert.True(t, proto.Equal(want.Spans[0], gotOc[0].Spans[0]))
+	assert.EqualValues(t, want.Spans[0], gotOc[0].Spans[0])
```

Comment:
Why was this change needed?

Reply:
Because this shows diffs between them in the failure, not just fail true != false :)